static/app/components/deployBadge.tsx

```diff
@@ -1,5 +1,5 @@
+import Tag from 'sentry/components/badge/tag';
```

Comment:
is there a reason we keep this `deployBadge`? i noticed you made another copy under `/badge/deployBadge` as well

Reply:
good catch, this should be gone.